recipes/asciidoc/meta.yaml

```diff
@@ -0,0 +1,52 @@
+
+{% set name = "asciidoc" %}
+{% set version = "8.6.10" %}
+{% set sha256 = "9e52f8578d891beaef25730a92a6e723596ddbd07bfe0d2a56486fcf63a0b983" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/{{ name }}/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py3k]
```

Comment:
Is there an install step somewhere? Seem to be missing `script` or `build.sh`.

Reply:
or probably use an entry point to the main script?

Reply:
I think that the correct approach could be:
* create a patch introducing a main() function in the main script and in `setup.py`
* create an entry point there

what do you think @jakirkham? this would also avoid  to copy the `source_files` as above in the test section

Reply:
but there is no `setup.py`... 😞